code/__DEFINES/misc.dm

```diff
@@ -255,10 +255,11 @@
                             0, 0, 1, 0,\
                             0, 0, 0, 1)
 
-#define MATRIX_GREYSCALE list(0.3, 0.3, 0.3, 0,\
-                              0.3, 0.3, 0.3, 0,\
-                              0.3, 0.3, 0.3, 0,\
-                              0,   0,   0,   1)
+#define MATRIX_GREYSCALE list(0.33, 0.33, 0.33, 0,\
```

Comment:
**TLDR:** A matrix and color matrix operate the same way but are not equal since "[matrices] represented by the /matrix datum [are] used to transform 2D coordinates" while "[color matrices are] used to transform colors" and thus, both matrices contain different parameters to do different transformations. Regular matrices are 3x3 and colour matrices are 5x5 according to the DM reference page [here](**url**)

It eliminated the difference between color_view (@ noir glasses) and usr.client.color

When I was originally testing to see why color_view != usr.client.color, I used View Variables to see, after activating noir mode on the shades as a human, what the lists looked like for color_view (navigates to the glasses in my mob's view variables) and my mob's client's color (navigated to client in my mob's view variables)

At first, before I made this change, color_view was a 16 entry list however my mob's color list was 20 entries, the last 4 all zeroes. That meant that color_view couldn't possibly equal usr.client.color because the `.len`s were different (although I couldn't use .len to test that on either because it turns out they're both just var/whatever instead of var/list/whatever).

So I made this change and referenced VG to see how they managed it (b/c I remember them adopting it some time ago as well) and they had greyscale defined this way, too. They also implemented the difflist proc some place (I wrote all this in the PR main comment) and so I decided to try it as well.

However, difflist still thought the lists were different (this is after I made this color-matrix a 20-entry 4x5) for whatever reason despite my verifying that they both looked the exact same and contained the exact same 20 values via View Variables. I had a hunch it had to do with difflist returning an empty list that somehow had a `.len`, so I rolled the dice, made that change too, and it sorted out my problems.